src/OPLS.js

```diff
@@ -47,8 +48,9 @@ export class OPLS {
     } else if (typeof labels[0] === 'string') {
       // non-numeric labels: OPLS-DA is used
       this.mode = 'discriminantAnalysis';
-      group = labels;
-      throw new Error('discriminant analysis is not yet supported');
+      const dummyY = Matrix.checkMatrix(createDummyY(labels)).transpose();
+      group = Matrix.checkMatrix(dummyY);
+      // group = labels;
```

Comment:
```suggestion
      group = Matrix.checkMatrix(createDummyY(labels)).transpose();
```

Not sure why the intermediate variable